src/main/fc/cli.c

```diff
@@ -3154,21 +3154,23 @@ static void cliTimer(char *cmdline)
     int timerChannelIndex = 0;
     
     pch = strtok_r(cmdline, " ", &saveptr);
-    for (int i = 0; i < TIMER_CHANNEL_COUNT; i++) {
-        const uint8_t timer = TIMER_TAG_INDEX(timerTags[i]);
-        const uint8_t channel = TIMER_TAG_CHANNEL_INDEX(timerTags[i]);    
+    if (pch) {
+        for (int i = 0; i < TIMER_CHANNEL_COUNT; i++) {
+            const uint8_t timer = TIMER_TAG_INDEX(timerTags[i]);
+            const uint8_t channel = TIMER_TAG_CHANNEL_INDEX(timerTags[i]);    
 
-        for(inverted = false; !inverted; inverted = true) {
-            tfp_sprintf(buffer, timChformat, timer, channel, inverted ? "N" : "");
+            for(inverted = false; !inverted; inverted = true) {
```

Comment:
Sorry, I was wrong, this won't work (look condition is tested after update, so body is executed only once)

One alternative is:
```
for (j = 0; j <= 1; j++) {
   bool inverted = j;
```